repo/packages/M/marathon/0/config.json

```diff
@@ -0,0 +1,378 @@
+{
+  "additionalProperties" : false,
+  "description" : "Marathon universe properties",
+  "properties" : {
+    "marathon" : {
+      "additionalProperties" : false,
+      "description" : "Marathon app configuration properties.",
+      "properties" : {
+        "id" : {
+          "description" : "The id of this marathon application name.",
+          "type" : "string"
+        },
+        "cpus" : {
+          "default" : 2,
+          "description" : "CPU shares to allocate to each Marathon instance.",
+          "minimum" : 0,
+          "type" : "number"
+        },
+        "mem" : {
+          "default" : 1024,
+          "description" : "Memory (MB) to allocate to each Marathon task.",
```

Comment:
I would use `instance` here instead of `task` to be consistent with the previous property.